src/app/modules/group/services/group-datasource.service.ts

```diff
@@ -22,15 +27,19 @@ export class GroupDataSource implements OnDestroy {
   state$ = this.fetchOperation.pipe(
     // switchMap does cancel the previous ongoing processing if a new one comes
     // on new fetch operation to be done: set "fetching" state and fetch the data which will result in a ready or error state
-    switchMap(route => this.getGroupByIdService.get(route.id).pipe(
-      map(group => ({ route, group })),
+    switchMap(route => forkJoin({
+      group: this.getGroupByIdService.get(route.id),
+      navigation: this.groupNavigationService.getGroupNavigation(route.id),
```

Comment:
We do not need/want to wait for the nav data to display the group page, so either we need to use `buildUp`, but it means the 2nd fetch will not start before the first one has responded, or we "buildUp" the answer without waiting, so using the `zip` operator... but without emitting the nav while the get-group-by-id has not been received... we need a "buildUpParallel" operator 😁
Ok, we can do this later...